consensus/poet/core/tests/test_consensus/test_poet_block_publisher.py

```diff
@@ -674,3 +675,233 @@ def test_block_publisher_success_case(
         self.assertTrue(
             block_publisher.initialize_block(
                 block_header=mock_block.header))
+
+    @patch('sawtooth_poet.poet_consensus.wait_timer.WaitTimer')
```

Comment:
You can already do this with `@mock.patch`.  Either change all `@mock.patch` to `@patch` or make this one to be `@mock.patch`.  You choose, but stay consistent.